src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralSummaryRepositoryImpl.kt

```diff
@@ -50,8 +50,7 @@ class ReferralSummaryRepositoryImpl : ReferralSummaryRepository {
 		  r.sent_at is not null
 		  and ( dfc.prime_provider_id in :serviceProviders or dfcsc.subcontractor_provider_id in :serviceProviders )
       and not (r.concluded_At is not null and r.end_Requested_At is not null and eosr.id is null)
-      $dashboardRestrictionCriteria
-) a where assigned_at_desc_seq = 1"""
+) a where assigned_at_desc_seq = 1 $dashboardRestrictionCriteria"""
```

Comment:
to note, `referral_assignments` has a `superseded` flag (still needs to be backfilled) which should allow for simpler conditions -- ie. `assignment.superseded = false` would mean it's the latest one

Reply:
or perhaps, a better pattern would be introducing `previous_version_id` to create a linked list for _any_ versioned entry

Reply:
Good points. This PR has the least divergence with the existing native SQL; but perhaps we should have this in for consideration for the pagination jpa specification route?